doc/sphinx/dials_scale_user_guide.rst

```diff
@@ -19,60 +21,98 @@ The user is also advised to familiarise themselves with the standard program
-  bad per-image R-merge values as result of radiation damage. Use multiple
-  commands like this to exclude multiple ranges. The syntax is "a:b:c",
-  where a is the experiment number (starting at :samp:`0`), b is the initial image
-  to exclude and c is the final image to exclude.
-- :samp:`exclude_datasets="10 50 79"`  Go all out and remove whole datasets, based on
+**Cutting back data**
+After inspecting the statistics in the :samp:`dials.scale.html` file, such as
+the R-merge vs batch plot, it is often the case that not all of the data are
+suitable for merging, perhaps due to radiation damage or nonisomorphism.
+This can be the case within a  single sweep or across multiple sweeps in a
+multi-sweep/multi-crystal experiment. These are example options to use:
+
+- :samp:`d_min=2.0`  Applies a resolution cutoff at the given resolution (in Angstrom).
+- :samp:`exclude_images="100:120"`  Removes a section of images for a single
+  sweep dataset. Multiple commands like this can be used to exclude multiple ranges.
+  In the case of multiple-sweeps, one must also provide the experiment id that
+  the exclusion should apply to, with the syntax :samp:`exclude_images="a:b:c"`
+  where `a` is the experiment id (a number starting at :samp:`0`), `b` is the initial
+  image to exclude and `c` is the final image to exclude.
+- :samp:`exclude_datasets="10 50 79"`  Removes whole datasets, based on
   the dataset number; useful for large multi-crystal datasets.
 
-**Eyes on the prize - getting those MTZ files**.
-You're confident that this job is a good one and are already daydreaming about
-refining a high-quality structure. Use these options to get an mtz file from
-your final run of dials.scale without the extra step of using the full
-dials.export/dials.merge programs (which may have more extensive options):
+**Anomalous data**
+During scaling, the option :samp:`anomalous=[True|False]` determines whether
+anomalous pairs (I+/I-) are combined during scaling model minimisation and outlier
+rejection. By default, :samp:`anomalous=False`, which is a suitable scaling protocol
+in DIALS for data with some anomalous signal, however for strongly anomalous data,
+the anomalous signal strength may be enhanced when scaling with :samp:`anomalous=True`
+
+**Controlling the absorption correction**
+The default physical scaling model applies a relative absorption correction based
+on the incoming and outgoing scattering vectors (this accounts for the relative
+difference in absorption for different scattering paths through the crystal,
+rather than absolute absorption of the beam by the crystal). This correction is
+constrained, and the level of constraint and parameterisation can be changed
+with the option :samp:`absorption_level=[low|medium|high]`. This aims to give
+relative absorption corrections of around 1%, 5% and 25%, but will depend on
+the dataset. To see the extent of the correction, check the 'scaling models'
```

Comment:
```suggestion
the dataset. To see the extent of the correction, check the 'Scaling Models'
```
I'm guessing (haven't actually checked) that this is capitalised in the html report.